copter/source/docs/traditional-helicopter-tips.rst

```diff
@@ -0,0 +1,33 @@
+
+===========================
+Traditional Helicopter Tips
+===========================
+
+- Be sure to follow the wiki instructions, step by step!
+- Be sure the CG is in line with the main rotor shaft
+- Blade tracking must be within a quarter blade thickness or better to avoid excessive vibration
+- Vibration is the arch enemy of Helicopter autopilots!
+   - Internal Combustion Engines and Turbines can created high frequency vibrations that do not appear in the VIBE logs and can be aliased to low frequencies which can impact EKF and Attitude control (see below for ground tests to check for this). This also often occurs in electric helicopters.
+   - The autopilot should be well isolated for vibration. Using autopilots with internal IMU vibration damping and vibration reducing mounts is recommended.
+- Traditional Helicopter *is* capable of inverted flight and using full negative collective ranges.
+
+Checking for High Frequency Vibration
+=====================================
+
+Symptoms of high frequency vibration can be that the attitude estimate becomes incorrect causing it to lean significantly even though the pilot is commanding level flight in stabilized modes. In the extreme, it can result in an :ref:`ekf-inav-failsafe` or even a crash. Excessive vibration can also make altitude hold imprecise or even produce uncontrollable climbs/descents.
+
+One method of testing this, is to secure the helicopter to the ground, and run up the main rotor, watching the artificial horizon in the ground control station. If it moves away from level, you probably have a vibration issue which needs to be addressed. Testing should be done at various collective levels and rotor speeds. Be careful not to over-stress the helicopter with collective extremes.
+
+Setup for ground test:
+
+#. Secure vehicle to the ground.
+#. Temporarily reduce Roll, Pitch, and Yaw  P,I,D,and FF PIDs to zero. Be sure to restore them after this test!
+#. Set :ref:`ACRO_OPTIONS<ACRO_OPTIONS>` = 2, temporarily, if not being used already.
+#. Change to ACRO mode. 
+#. Arm and engage motor interlock, allowing the vehicle to spool up at 0 degree pitch, watching for any change from level in the GSC of the artificial horizon. Any significant tilt indicates noise is disrupting the attitude estimate and should be investigated and eliminated.
+#. Change the target main rotor speed a bit above and below the nominal target using the :ref:`H_RSC_SETPOINT<H_RSC_SETPOINT>` (or TX throttle curve if using Passthru mode) and repeat.
+#. Re-check the blade tracking also while doing this test.
+
+Be sure to also eliminate the possibility of low frequency vibrations causing the issue. Unlike high frequency vibrations which cannot be eliminated by software filtering, low frequency vibrations can be improved using ArduPilot's notch filtering feature. See :ref:`common-measuring-vibration` and :ref:`Helicopter Dynamic Notch Filter Setup<common-imu-notch-filtering-helicopter-setup>`.
```

Comment:
I would point out in this first sentence that these vibrations are almost always multiples of the rotor speed and the harmonic notch is well suited for attenuating these vibrations.  Also I'm not a big fan of your use of low and high frequency.  My definition of low is not your definition of low.  I would rather you point out that high would be frequencies above the sampling rate of the IMU (~1k to 8k hz depending on IMU) which can't be removed by software filtering

Reply:
fixed